data/projects.yml

```
@@ -0,0 +1,16 @@
+title: Projects
+introduction: |
+  These are the open source projects which are officially hosted by the
+  OpenRail Association. They have been accepted through the incubation process.
+  If you would like to suggest a new project to the OpenRail Association please
+  have a look at the questionnaire for new projects.
+
+listclass: style1 medium less-bottom-margin less-top-margin
+
+projects:
+  - publiccode: https://raw.githubusercontent.com/OpenRailAssociation/landscape/main/data/openrail/osrd.yml
+  - publiccode: https://github.com/OpenRailAssociation/landscape/raw/main/data/openrail/rcm-oss.yml
+  - publiccode: https://github.com/OpenRailAssociation/landscape/raw/main/data/openrail/dac-migration-dss.yml
+  - publiccode: https://github.com/OpenRailAssociation/landscape/raw/main/data/openrail/nge.yml
+  - publiccode: https://github.com/OpenRailAssociation/landscape/raw/main/data/openrail/liblrs.yml
```

Use final links directly, spares the build to follow a 302 redirect:

```suggestion
  - publiccode: https://raw.githubusercontent.com/OpenRailAssociation/landscape/main/data/openrail/osrd.yml
  - publiccode: https://raw.githubusercontent.com/OpenRailAssociation/landscape/main/data/openrail/rcm-oss.yml
  - publiccode: https://raw.githubusercontent.com/OpenRailAssociation/landscape/main/data/openrail/dac-migration-dss.yml
  - publiccode: https://raw.githubusercontent.com/OpenRailAssociation/landscape/main/data/openrail/nge.yml
  - publiccode: https://raw.githubusercontent.com/OpenRailAssociation/landscape/main/data/openrail/liblrs.yml
```

---

Also see #31, this should probably be integrated here